### PR TITLE
13746 Fix issue with staff

### PR DESCRIPTION
**Pull request by ChasBoucher**
*Issue #:* /bcgov/entity#13746

*Description of changes:*

- Added logic to ensure that existing Dashboard for Mhr Staff is not affected by current ticket.. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts [hunk 1]
```diff
@@ -199,9 +201,11 @@ export const useNewMhrRegistration = () => {
 
   const fetchMhRegistrations = async (): Promise<void> => {
     const draftFilings = await getMhrDrafts()
-    const myMhrHistory = await mhrRegistrationHistory(true)
-    const filteredMhrHistory = addHistoryDraftsToMhr(myMhrHistory, draftFilings)
-    setMhrTableHistory([...filteredMhrHistory])
+    const myMhrHistory = await mhrRegistrationHistory(isRoleQualifiedSupplier.value)
+    if (isRoleQualifiedSupplier.value === true) {
```
Comment (cameron-eyds) [inline ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts]: Just curious, does `isRoleQualifiedSupplier.value` resolve to `true` or `false` accurately as well?

**Review by cameron-eyds — APPROVED** [review]
Looks good, just a minor comment.

File: ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts (hunk 1 above)
Comment (ChasBoucher) [reply ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts]: Yes

Comment (cameron-eyds) [reply ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts]: So could get away with `if (isRoleQualifiedSupplier.value) {...}`?

File: ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts [hunk 2]
```diff
@@ -199,9 +201,11 @@ export const useNewMhrRegistration = () => {
...
+      const filteredMhrHistory = addHistoryDraftsToMhr(myMhrHistory, draftFilings)
+      setMhrTableHistory([...filteredMhrHistory])
```
Comment (dimak1) [inline ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts]: We can omit the spread operator because `filteredMhrHistory` is already an array.

**Review by dimak1 — APPROVED** [review]
Looks good to me. Just few comments from me and Cameron.

File: ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts (hunk 1 above)
Comment (ChasBoucher) [reply ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts]: Chaned..

File: ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts (hunk 2 above)
Comment (ChasBoucher) [reply ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts]: Made the change..

**Pull request merged**